PerformanceTesting/Scripts/EndUserJourneys/LandlordSendsMessageAcceptBooking.js

```diff
@@ -0,0 +1,289 @@
+import { check } from "k6";
+import http from "k6/http";
+import { group } from "k6";
```

Comment:
imports from the same package can be grouped together.

```suggestion
import { check, group } from "k6";
import http from "k6/http";
```

Reply:
done, thanks!